src/Core/src/Platform/Windows/RootNavigationView.cs

```diff
@@ -329,7 +329,7 @@ void UpdateFlyoutPaneSize()
 			if (PaneContentGrid == null)
 				return;
 
-			var newSize = new Size(OpenPaneLength, PaneContentGrid.ActualHeight - PaneContentGrid.RowDefinitions[1].Height.Value);
+			var newSize = new Size(OpenPaneLength, Math.Max(PaneContentGrid.ActualHeight - PaneContentGrid.RowDefinitions[1].Height.Value, 0));
```

Comment:
Is there any reason to let this continue if we know the layout hasn't occurred yet? 

would something like this work?

This code is going to have to run a second time with correct values once the flyout is opened so it seems like we shouldn't really set anything quite yet. 

```C#
			var newHeight = PaneContentGrid.ActualHeight - PaneContentGrid.RowDefinitions[1].Height.Value;
			if (newHeight < 0)
				return;

			var newSize = new Size(OpenPaneLength, newHeight);
```

Reply:
Yeah, I was debating whether to go with the early return or the more minimal change. I prefer the early return myself, since I think it's technically better, but ironically thought the more minimal change was likely to be accepted. At the end of the day I just want my app not to crash, so I'm happy to change it.